src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/Interfaces/IVsTestConsoleWrapper.cs

```diff
@@ -119,6 +120,14 @@ public interface IVsTestConsoleWrapper
         /// <param name="customTestHostLauncher">Custom test host launcher for the run.</param>
         void RunTestsWithCustomTestHost(IEnumerable<TestCase> testCases, string runSettings, TestPlatformOptions options, ITestRunEventsHandler testRunEventsHandler, ITestHostLauncher customTestHostLauncher);
 
+        /// <summary>
+        /// Provides back all attachments to TestPlatform for additional processing (for example merging)
+        /// </summary>
+        /// <param name="attachments">Collection of attachments</param>
+        /// <param name="multiTestRunFinalizationCompleteEventsHandler">EventHandler to receive session complete event</param>
+        /// <param name="cancellationToken">Cancellation token</param>
+        Task FinalizeMultiTestRunAsync(ICollection<AttachmentSet> attachments, IMultiTestRunFinalizationEventsHandler multiTestRunFinalizationCompleteEventsHandler, CancellationToken cancellationToken);
```

Comment:
Tagging @nohwnd  and @cvpoienaru on the changes in this file and its async equivalent. The assumption is that external consumers do not implement this interface so its ok to change it without introducing a IVSTestConsoleWrapper2. Do you have any concerns?

Reply:
I don't have any. All usages of this interface I saw were consuming it, not implementing it.

Reply:
I didn't find any implementations of it. Resolving this.